cfme/tests/test_appliance.py

```diff
@@ -24,7 +24,6 @@
     'rhn-check',
     'rhnlib',
 ])
-@test_requirements.appliance
```

Comment:
I am curious, why are we deleting all of the "@test_requirements.appliance" lines for tests in this file. After the deletion, most of the tests don't have a @test_requirements marker and I see the casecomponent in the Polarion docblock is set to Appliance?

Reply:
I guess above is because there is no "appliance" section in test_requirements.py?

Reply:
because the marker is included at the module level, L13 above.

Reply:
@prichard77 @mshriver moved it to the module pytestmark